bridge/core/html/parser/html_parser.cc

```diff
@@ -92,6 +93,24 @@ void HTMLParser::traverseHTML(Node* root_node, GumboNode* node) {
       } else if (child->type == GUMBO_NODE_TEXT) {
         auto* text = context->document()->createTextNode(AtomicString(ctx, child->v.text.text), ASSERT_NO_EXCEPTION());
         root_container->AppendChild(text);
+      } else if (child->type == GUMBO_NODE_WHITESPACE) {
```

Comment:
Why not check GUMBO_NODE_COMMENT also?

Reply:
it only to judge the empty: if the element only have the blank space ,  i will add textnode to  this element. or do nothing, at last  empty judgement code is to judge the element  has sub element